packages/flutter/lib/src/widgets/selection_container.dart

```diff
@@ -179,7 +179,21 @@ class _SelectionContainerState extends State<SelectionContainer> with Selectable
   @override
   SelectedContent? getSelectedContent() {
     assert(!widget._disabled);
-    return widget.delegate!.getSelectedContent();
+    final SelectedContent? content = widget.delegate!.getSelectedContent();
+    if (content == null) return null;
```

Comment:
```suggestion
    if (content == null) {
      return null;
    }
```

Control flow requires brackets.

Reply:
I added your suggestion. However, there is no specification for this in the [Flutter style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#overview) which defers to the [Dart style guide](https://dart.dev/effective-dart/style#do-use-curly-braces-for-all-flow-control-statements) for cases not covered in the former. 

I've linked the rule which contains the following exception:
`Exception: When you have an if statement with no else clause and the whole if statement fits on one line, you can omit the braces if you prefer`

Should the Flutter style guide be updated to reflect that this exception is not allowed?

Reply:
There was a failing check in the Linux analyze Task, hence why I suggested it.

Reply:
> There was a failing check in the Linux analyze Task, hence why I suggested it.

Of course, however is this something I should add to the style guide as well?

Reply:
For context, I saw this check in the `Linux analyze` task for commit https://github.com/flutter/flutter/commit/4865ea3c94a77b2979aee5cf5de7d9eb7bd316ad

https://github.com/flutter/flutter/runs/30458026970 reported `Statement should be on a separate line packages/flutter/lib/src/widgets/selection_container.dart:183:26 always_put_control_body_on_new_line` for line 183

I don't think you need to add it to the style guide.